# Hypothesis testing

### Step-1  :- Hypothesis Statements

Null Hypothesis Ho : The operating cost for a franchise in a week (W) = $1000 + $5X

Alternate Hypothesis H1 : The operating cost for a franchise in a week (W) > $1000 + $5X

### Step-2 :- Select Test Static

There are 25 samples.We can go for  t-test for sample size < 30.


t=((sample mean - population mean) / (standard deviation/sqrt(sample size)))

Given data,

sample mean(x)=3050

Population mean (u)= 1000+5x (x=600) = 4000

sample size (n) = 25

standard deviation = 5*25 = 125


In [2]:
import numpy as np
Tcal=(3050-4000)/(125/np.sqrt(25))
Tcal

-38.0

It is a one tailed test.So we dont need to take absolute value for t calculated.

### Step-3 :- Determine Significance Level

level of significance alpha(a)= 5 % or 0.05.

degree of freedom (df) = 25-1= 24

In [8]:
import scipy.stats as stats
stats.t.ppf(0.975,24)

2.0638985616280205

### Step-4 :- Make decision

Tcal= -38 is very less value compared to T critical value (2.06).So we are failed to reject null hypothesis statement.

### Step-5 :- Conclusion

Based on the T-test ,we don't have any evidence to reject the Null Hypothesis.There is no strong evidence to support the restaurant owners claim that the weekly operating costs are higher than the model suggests.